From: Qingqing <wangyiqing@kylinos.cn>
Date: Fri, 17 Jul 2026 08:03:34 +0000
Subject: Translated using Weblate (Korean)

Currently translated at 98.4% (62 of 63 strings)

Translation: openkylin-huanghe/qt5-ukui-platformtheme
Translate-URL: http://weblate.openkylin.top/projects/openkylin-huanghe/qt5-ukui-platformtheme/ko/
---
 translations/qt5-ukui-platformtheme_ar.ts | 124 +++++++++++++++---------------
 translations/qt5-ukui-platformtheme_ko.ts | 124 +++++++++++++++---------------
 2 files changed, 124 insertions(+), 124 deletions(-)

diff --git a/translations/qt5-ukui-platformtheme_ar.ts b/translations/qt5-ukui-platformtheme_ar.ts
index 39505b7..113ecca 100644
--- a/translations/qt5-ukui-platformtheme_ar.ts
+++ b/translations/qt5-ukui-platformtheme_ar.ts
@@ -9,53 +9,53 @@
         <location filename="../test/mps-style-application/mainwindow.ui" line="14"/>
         <location filename="../test/system-settings/mainwindow.ui" line="14"/>
         <source>MainWindow</source>
-        <translation type="unfinished"></translation>
+        <translation>النافذة الرئيسية</translation>
     </message>
     <message>
         <location filename="../test/filedialog/mainwindow.ui" line="30"/>
         <source>test open</source>
-        <translation type="unfinished"></translation>
+        <translation>فتح الاختبار</translation>
     </message>
     <message>
         <location filename="../test/filedialog/mainwindow.ui" line="37"/>
         <source>directory</source>
-        <translation type="unfinished"></translation>
+        <translation>دليل</translation>
     </message>
     <message>
         <location filename="../test/filedialog/mainwindow.ui" line="47"/>
         <source>selected uri</source>
-        <translation type="unfinished"></translation>
+        <translation>الرابط المحدد</translation>
     </message>
     <message>
         <location filename="../test/filedialog/mainwindow.ui" line="54"/>
         <source>test show</source>
-        <translation type="unfinished"></translation>
+        <translation>عرض الاختبار</translation>
     </message>
     <message>
         <location filename="../test/filedialog/mainwindow.ui" line="64"/>
         <source>test exec</source>
-        <translation type="unfinished"></translation>
+        <translation>تنفيذ الاختبار</translation>
     </message>
     <message>
         <location filename="../test/filedialog/mainwindow.ui" line="71"/>
         <source>test save</source>
-        <translation type="unfinished"></translation>
+        <translation>حفظ الاختبار</translation>
     </message>
     <message>
         <location filename="../test/filedialog/mainwindow.ui" line="78"/>
         <source>test static open</source>
-        <translation type="unfinished"></translation>
+        <translation>اختبار فتح ثابت</translation>
     </message>
     <message>
         <location filename="../test/highlighted-icon-button/mainwindow.ui" line="30"/>
         <location filename="../test/mps-style-application/mainwindow.ui" line="36"/>
         <source>PushButton</source>
-        <translation type="unfinished"></translation>
+        <translation>زر الضغط</translation>
     </message>
     <message>
         <location filename="../test/highlighted-icon-button/mainwindow.ui" line="47"/>
         <source>use auto highlight icon</source>
-        <translation type="unfinished"></translation>
+        <translation>استخدام رمز التمييز التلقائي</translation>
     </message>
     <message>
         <location filename="../test/highlighted-icon-button/mainwindow.ui" line="54"/>
@@ -66,37 +66,37 @@
     <message>
         <location filename="../test/highlighted-icon-button/mainwindow.ui" line="72"/>
         <source>highlightOnly</source>
-        <translation type="unfinished"></translation>
+        <translation>تمييز فقط</translation>
     </message>
     <message>
         <location filename="../test/highlighted-icon-button/mainwindow.ui" line="77"/>
         <source>bothDefaultAndHighlight</source>
-        <translation type="unfinished"></translation>
+        <translation>كلا الافتراضي والإبراز</translation>
     </message>
     <message>
         <location filename="../test/mps-style-application/mainwindow.ui" line="50"/>
         <source>RadioButton</source>
-        <translation type="unfinished"></translation>
+        <translation>زر الراديو</translation>
     </message>
     <message>
         <location filename="../test/system-settings/mainwindow.ui" line="21"/>
         <source>style</source>
-        <translation type="unfinished"></translation>
+        <translation>نمط</translation>
     </message>
     <message>
         <location filename="../test/system-settings/mainwindow.ui" line="31"/>
         <source>icon</source>
-        <translation type="unfinished"></translation>
+        <translation>رمز</translation>
     </message>
     <message>
         <location filename="../test/system-settings/mainwindow.ui" line="41"/>
         <source>menu opacity</source>
-        <translation type="unfinished"></translation>
+        <translation>عتامة القائمة</translation>
     </message>
     <message>
         <location filename="../test/system-settings/mainwindow.ui" line="55"/>
         <source>font</source>
-        <translation type="unfinished"></translation>
+        <translation>الخط</translation>
     </message>
 </context>
 <context>
@@ -104,7 +104,7 @@
     <message>
         <location filename="../qt5-ukui-platformtheme/widget/messagebox/message-box.cpp" line="97"/>
         <source>Close</source>
-        <translation type="unfinished"></translation>
+        <translation>إغلاق</translation>
     </message>
 </context>
 <context>
@@ -112,12 +112,12 @@
     <message>
         <location filename="../qt5-ukui-platformtheme/widget/messagebox/message-box.h" line="272"/>
         <source>Executable &apos;%1&apos; requires Qt %2, found Qt %3.</source>
-        <translation type="unfinished"></translation>
+        <translation>الملف التنفيذي &quot;%1&quot; يتطلب الكمية %2، تم العثور على الكمية %3.</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-platformtheme/widget/messagebox/message-box.h" line="274"/>
         <source>Incompatible Qt Library Error</source>
-        <translation type="unfinished"></translation>
+        <translation>خطأ في مكتبة Qt غير المتوافقة</translation>
     </message>
 </context>
 <context>
@@ -126,7 +126,7 @@
         <location filename="../qt5-ukui-platformtheme/widget/messagebox/message-box.cpp" line="362"/>
         <location filename="../qt5-ukui-platformtheme/widget/messagebox/message-box.cpp" line="1090"/>
         <source>OK</source>
-        <translation type="unfinished"></translation>
+        <translation>تأكيد</translation>
     </message>
 </context>
 <context>
@@ -136,13 +136,13 @@
         <location filename="../qt5-ukui-platformtheme/widget/messagebox/message-box.cpp" line="881"/>
         <location filename="../qt5-ukui-platformtheme/widget/messagebox/message-box.cpp" line="1517"/>
         <source>Show Details...</source>
-        <translation type="unfinished"></translation>
+        <translation>عرض التفاصيل...</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-platformtheme/widget/messagebox/message-box.cpp" line="881"/>
         <location filename="../qt5-ukui-platformtheme/widget/messagebox/message-box.cpp" line="1517"/>
         <source>Hide Details...</source>
-        <translation type="unfinished"></translation>
+        <translation>إخفاء التفاصيل...</translation>
     </message>
 </context>
 <context>
@@ -150,57 +150,57 @@
     <message>
         <location filename="../qt5-ukui-filedialog/ui_kyfiledialog.cpp" line="259"/>
         <source>File Name</source>
-        <translation type="unfinished"></translation>
+        <translation>اسم الملف</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/ui_kyfiledialog.cpp" line="263"/>
         <source>Modified Date</source>
-        <translation type="unfinished"></translation>
+        <translation>تاريخ التعديل</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/ui_kyfiledialog.cpp" line="267"/>
         <source>File Type</source>
-        <translation type="unfinished"></translation>
+        <translation>نوع الملف</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/ui_kyfiledialog.cpp" line="271"/>
         <source>File Size</source>
-        <translation type="unfinished"></translation>
+        <translation>حجم الملف</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/ui_kyfiledialog.cpp" line="275"/>
         <source>Original Path</source>
-        <translation type="unfinished"></translation>
+        <translation>المسار الأصلي</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/ui_kyfiledialog.cpp" line="284"/>
         <source>Descending</source>
-        <translation type="unfinished"></translation>
+        <translation>تنازلي</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/ui_kyfiledialog.cpp" line="289"/>
         <source>Ascending</source>
-        <translation type="unfinished"></translation>
+        <translation>تصاعدي</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/ui_kyfiledialog.cpp" line="295"/>
         <source>Use global sorting</source>
-        <translation type="unfinished"></translation>
+        <translation>استخدام الترتيب العام</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/ui_kyfiledialog.cpp" line="315"/>
         <source>List View</source>
-        <translation type="unfinished"></translation>
+        <translation>عرض القائمة</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/ui_kyfiledialog.cpp" line="316"/>
         <source>Icon View</source>
-        <translation type="unfinished"></translation>
+        <translation>عرض الرموز</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-platformtheme/widget/messagebox/message-box.cpp" line="730"/>
         <source>Close</source>
-        <translation type="unfinished"></translation>
+        <translation>إغلاق</translation>
     </message>
 </context>
 <context>
@@ -208,12 +208,12 @@
     <message>
         <location filename="../libqt5-ukui-style/animations/tabwidget/ukui-tabwidget-default-slide-animator-factory.h" line="50"/>
         <source>Default Slide</source>
-        <translation type="unfinished"></translation>
+        <translation>الشرائح الافتراضية</translation>
     </message>
     <message>
         <location filename="../libqt5-ukui-style/animations/tabwidget/ukui-tabwidget-default-slide-animator-factory.h" line="51"/>
         <source>Let tab widget switch with a slide animation.</source>
-        <translation type="unfinished"></translation>
+        <translation>جعل عنصر التبويب يتحول إلى تحريك شرائح.</translation>
     </message>
 </context>
 <context>
@@ -221,17 +221,17 @@
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="2706"/>
         <source>Open File</source>
-        <translation type="unfinished"></translation>
+        <translation>فتح</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="2707"/>
         <source>Save File</source>
-        <translation type="unfinished"></translation>
+        <translation>حفظ</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="2720"/>
         <source>All Files (*)</source>
-        <translation type="unfinished"></translation>
+        <translation>الكل(*)</translation>
     </message>
 </context>
 <context>
@@ -239,123 +239,123 @@
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="221"/>
         <source>Go Back</source>
-        <translation type="unfinished"></translation>
+        <translation>رجوع</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="228"/>
         <source>Go Forward</source>
-        <translation type="unfinished"></translation>
+        <translation>للأمام</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="234"/>
         <source>Cd Up</source>
-        <translation type="unfinished"></translation>
+        <translation>أعلى</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="241"/>
         <source>Search</source>
-        <translation type="unfinished"></translation>
+        <translation>بحث</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="249"/>
         <source>View Type</source>
-        <translation type="unfinished"></translation>
+        <translation>نوع العرض</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="260"/>
         <source>Sort Type</source>
-        <translation type="unfinished"></translation>
+        <translation>نوع الترتيب</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="265"/>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="378"/>
         <source>Maximize</source>
-        <translation type="unfinished"></translation>
+        <translation>تكبير</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="279"/>
         <source>Close</source>
-        <translation type="unfinished"></translation>
+        <translation>إغلاق</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="375"/>
         <source>Restore</source>
-        <translation type="unfinished"></translation>
+        <translation>استعادة</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="987"/>
         <source>Name</source>
-        <translation type="unfinished"></translation>
+        <translation>اسم الملف</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="990"/>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="1578"/>
         <source>Open</source>
-        <translation type="unfinished"></translation>
+        <translation>فتح</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="991"/>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="1000"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>إلغاء</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="995"/>
         <source>Save as</source>
-        <translation type="unfinished"></translation>
+        <translation>حفظ باسم</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="997"/>
         <source>New Folder</source>
-        <translation type="unfinished"></translation>
+        <translation>مجلد جديد</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="999"/>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="1582"/>
         <source>Save</source>
-        <translation type="unfinished"></translation>
+        <translation>حفظ</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="1035"/>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="1036"/>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="1038"/>
         <source>Directories</source>
-        <translation type="unfinished"></translation>
+        <translation>الدليل</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="1227"/>
         <source>Warning</source>
-        <translation type="unfinished"></translation>
+        <translation>تحذير</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="1227"/>
         <source>exist, are you sure replace?</source>
-        <translation type="unfinished"></translation>
+        <translation>موجود بالفعل، هل تريد الاستبدال؟</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="1920"/>
         <source>NewFolder</source>
-        <translation type="unfinished"></translation>
+        <translation>مجلد جديد</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="2208"/>
         <source>Undo</source>
-        <translation type="unfinished"></translation>
+        <translation>تراجع</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="2215"/>
         <source>Redo</source>
-        <translation type="unfinished"></translation>
+        <translation>إعادة</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="2414"/>
         <source>warn</source>
-        <translation type="unfinished"></translation>
+        <translation>تحذير</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="2414"/>
         <source>This operation is not supported.</source>
-        <translation type="unfinished"></translation>
+        <translation>هذه العملية غير مدعومة.</translation>
     </message>
 </context>
 </TS>
diff --git a/translations/qt5-ukui-platformtheme_ko.ts b/translations/qt5-ukui-platformtheme_ko.ts
index 720f229..7426f56 100644
--- a/translations/qt5-ukui-platformtheme_ko.ts
+++ b/translations/qt5-ukui-platformtheme_ko.ts
@@ -9,53 +9,53 @@
         <location filename="../test/mps-style-application/mainwindow.ui" line="14"/>
         <location filename="../test/system-settings/mainwindow.ui" line="14"/>
         <source>MainWindow</source>
-        <translation type="unfinished"></translation>
+        <translation>주창</translation>
     </message>
     <message>
         <location filename="../test/filedialog/mainwindow.ui" line="30"/>
         <source>test open</source>
-        <translation type="unfinished"></translation>
+        <translation>시험 열기</translation>
     </message>
     <message>
         <location filename="../test/filedialog/mainwindow.ui" line="37"/>
         <source>directory</source>
-        <translation type="unfinished"></translation>
+        <translation>디렉토리</translation>
     </message>
     <message>
         <location filename="../test/filedialog/mainwindow.ui" line="47"/>
         <source>selected uri</source>
-        <translation type="unfinished"></translation>
+        <translation>선정된 uri</translation>
     </message>
     <message>
         <location filename="../test/filedialog/mainwindow.ui" line="54"/>
         <source>test show</source>
-        <translation type="unfinished"></translation>
+        <translation>시험 보기</translation>
     </message>
     <message>
         <location filename="../test/filedialog/mainwindow.ui" line="64"/>
         <source>test exec</source>
-        <translation type="unfinished"></translation>
+        <translation>시험 실행</translation>
     </message>
     <message>
         <location filename="../test/filedialog/mainwindow.ui" line="71"/>
         <source>test save</source>
-        <translation type="unfinished"></translation>
+        <translation>시험 저장</translation>
     </message>
     <message>
         <location filename="../test/filedialog/mainwindow.ui" line="78"/>
         <source>test static open</source>
-        <translation type="unfinished"></translation>
+        <translation>시험용 정적 열기</translation>
     </message>
     <message>
         <location filename="../test/highlighted-icon-button/mainwindow.ui" line="30"/>
         <location filename="../test/mps-style-application/mainwindow.ui" line="36"/>
         <source>PushButton</source>
-        <translation type="unfinished"></translation>
+        <translation>누름단추</translation>
     </message>
     <message>
         <location filename="../test/highlighted-icon-button/mainwindow.ui" line="47"/>
         <source>use auto highlight icon</source>
-        <translation type="unfinished"></translation>
+        <translation>자동 강조 아이콘 사용</translation>
     </message>
     <message>
         <location filename="../test/highlighted-icon-button/mainwindow.ui" line="54"/>
@@ -66,37 +66,37 @@
     <message>
         <location filename="../test/highlighted-icon-button/mainwindow.ui" line="72"/>
         <source>highlightOnly</source>
-        <translation type="unfinished"></translation>
+        <translation>강조전용</translation>
     </message>
     <message>
         <location filename="../test/highlighted-icon-button/mainwindow.ui" line="77"/>
         <source>bothDefaultAndHighlight</source>
-        <translation type="unfinished"></translation>
+        <translation>기본값과 강조 표시 모두</translation>
     </message>
     <message>
         <location filename="../test/mps-style-application/mainwindow.ui" line="50"/>
         <source>RadioButton</source>
-        <translation type="unfinished"></translation>
+        <translation>라디오단추</translation>
     </message>
     <message>
         <location filename="../test/system-settings/mainwindow.ui" line="21"/>
         <source>style</source>
-        <translation type="unfinished"></translation>
+        <translation>체계</translation>
     </message>
     <message>
         <location filename="../test/system-settings/mainwindow.ui" line="31"/>
         <source>icon</source>
-        <translation type="unfinished"></translation>
+        <translation>아이콘</translation>
     </message>
     <message>
         <location filename="../test/system-settings/mainwindow.ui" line="41"/>
         <source>menu opacity</source>
-        <translation type="unfinished"></translation>
+        <translation>메뉴 투명도</translation>
     </message>
     <message>
         <location filename="../test/system-settings/mainwindow.ui" line="55"/>
         <source>font</source>
-        <translation type="unfinished"></translation>
+        <translation>글꼴</translation>
     </message>
 </context>
 <context>
@@ -104,7 +104,7 @@
     <message>
         <location filename="../qt5-ukui-platformtheme/widget/messagebox/message-box.cpp" line="97"/>
         <source>Close</source>
-        <translation type="unfinished"></translation>
+        <translation>닫기</translation>
     </message>
 </context>
 <context>
@@ -112,12 +112,12 @@
     <message>
         <location filename="../qt5-ukui-platformtheme/widget/messagebox/message-box.h" line="272"/>
         <source>Executable &apos;%1&apos; requires Qt %2, found Qt %3.</source>
-        <translation type="unfinished"></translation>
+        <translation>실행파일 &quot;%1&quot;이(가) %2개 필요하지만 %3개만 찾았습니다.</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-platformtheme/widget/messagebox/message-box.h" line="274"/>
         <source>Incompatible Qt Library Error</source>
-        <translation type="unfinished"></translation>
+        <translation>호환되지 않는 Qt 라이브러리 오류</translation>
     </message>
 </context>
 <context>
@@ -126,7 +126,7 @@
         <location filename="../qt5-ukui-platformtheme/widget/messagebox/message-box.cpp" line="362"/>
         <location filename="../qt5-ukui-platformtheme/widget/messagebox/message-box.cpp" line="1090"/>
         <source>OK</source>
-        <translation type="unfinished"></translation>
+        <translation>확인</translation>
     </message>
 </context>
 <context>
@@ -136,13 +136,13 @@
         <location filename="../qt5-ukui-platformtheme/widget/messagebox/message-box.cpp" line="881"/>
         <location filename="../qt5-ukui-platformtheme/widget/messagebox/message-box.cpp" line="1517"/>
         <source>Show Details...</source>
-        <translation type="unfinished"></translation>
+        <translation>상세내용 보기…</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-platformtheme/widget/messagebox/message-box.cpp" line="881"/>
         <location filename="../qt5-ukui-platformtheme/widget/messagebox/message-box.cpp" line="1517"/>
         <source>Hide Details...</source>
-        <translation type="unfinished"></translation>
+        <translation>상세내용 숨기기…</translation>
     </message>
 </context>
 <context>
@@ -150,57 +150,57 @@
     <message>
         <location filename="../qt5-ukui-filedialog/ui_kyfiledialog.cpp" line="259"/>
         <source>File Name</source>
-        <translation type="unfinished"></translation>
+        <translation>파일명</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/ui_kyfiledialog.cpp" line="263"/>
         <source>Modified Date</source>
-        <translation type="unfinished"></translation>
+        <translation>수정날짜</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/ui_kyfiledialog.cpp" line="267"/>
         <source>File Type</source>
-        <translation type="unfinished"></translation>
+        <translation>파일 형식</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/ui_kyfiledialog.cpp" line="271"/>
         <source>File Size</source>
-        <translation type="unfinished"></translation>
+        <translation>파일크기</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/ui_kyfiledialog.cpp" line="275"/>
         <source>Original Path</source>
-        <translation type="unfinished"></translation>
+        <translation>원래 경로</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/ui_kyfiledialog.cpp" line="284"/>
         <source>Descending</source>
-        <translation type="unfinished"></translation>
+        <translation>내림차순</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/ui_kyfiledialog.cpp" line="289"/>
         <source>Ascending</source>
-        <translation type="unfinished"></translation>
+        <translation>오름차순</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/ui_kyfiledialog.cpp" line="295"/>
         <source>Use global sorting</source>
-        <translation type="unfinished"></translation>
+        <translation>전체렬순 사용</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/ui_kyfiledialog.cpp" line="315"/>
         <source>List View</source>
-        <translation type="unfinished"></translation>
+        <translation>렬표 보기</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/ui_kyfiledialog.cpp" line="316"/>
         <source>Icon View</source>
-        <translation type="unfinished"></translation>
+        <translation>도표보기</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-platformtheme/widget/messagebox/message-box.cpp" line="730"/>
         <source>Close</source>
-        <translation type="unfinished"></translation>
+        <translation>닫기</translation>
     </message>
 </context>
 <context>
@@ -208,12 +208,12 @@
     <message>
         <location filename="../libqt5-ukui-style/animations/tabwidget/ukui-tabwidget-default-slide-animator-factory.h" line="50"/>
         <source>Default Slide</source>
-        <translation type="unfinished"></translation>
+        <translation>기본슬라이드</translation>
     </message>
     <message>
         <location filename="../libqt5-ukui-style/animations/tabwidget/ukui-tabwidget-default-slide-animator-factory.h" line="51"/>
         <source>Let tab widget switch with a slide animation.</source>
-        <translation type="unfinished"></translation>
+        <translation>탭 위젯이 슬라이드 애니메이션으로 전환되도록 합니다.</translation>
     </message>
 </context>
 <context>
@@ -221,17 +221,17 @@
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="2706"/>
         <source>Open File</source>
-        <translation type="unfinished"></translation>
+        <translation>열기</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="2707"/>
         <source>Save File</source>
-        <translation type="unfinished"></translation>
+        <translation>저장</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="2720"/>
         <source>All Files (*)</source>
-        <translation type="unfinished"></translation>
+        <translation>모두(*)</translation>
     </message>
 </context>
 <context>
@@ -239,123 +239,123 @@
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="221"/>
         <source>Go Back</source>
-        <translation type="unfinished"></translation>
+        <translation>뒤로</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="228"/>
         <source>Go Forward</source>
-        <translation type="unfinished"></translation>
+        <translation>앞으로</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="234"/>
         <source>Cd Up</source>
-        <translation type="unfinished"></translation>
+        <translation>위로</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="241"/>
         <source>Search</source>
-        <translation type="unfinished"></translation>
+        <translation>검색</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="249"/>
         <source>View Type</source>
-        <translation type="unfinished"></translation>
+        <translation>보기방식</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="260"/>
         <source>Sort Type</source>
-        <translation type="unfinished"></translation>
+        <translation>렬순 종류</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="265"/>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="378"/>
         <source>Maximize</source>
-        <translation type="unfinished"></translation>
+        <translation>최대화</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="279"/>
         <source>Close</source>
-        <translation type="unfinished"></translation>
+        <translation>닫기</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="375"/>
         <source>Restore</source>
-        <translation type="unfinished"></translation>
+        <translation>복원</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="987"/>
         <source>Name</source>
-        <translation type="unfinished"></translation>
+        <translation>파일명</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="990"/>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="1578"/>
         <source>Open</source>
-        <translation type="unfinished"></translation>
+        <translation>열기</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="991"/>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="1000"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>취소</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="995"/>
         <source>Save as</source>
-        <translation type="unfinished"></translation>
+        <translation>다른 이름으로 저장</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="997"/>
         <source>New Folder</source>
-        <translation type="unfinished"></translation>
+        <translation>새 폴더</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="999"/>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="1582"/>
         <source>Save</source>
-        <translation type="unfinished"></translation>
+        <translation>저장</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="1035"/>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="1036"/>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="1038"/>
         <source>Directories</source>
-        <translation type="unfinished"></translation>
+        <translation>목록</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="1227"/>
         <source>Warning</source>
-        <translation type="unfinished"></translation>
+        <translation>경고</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="1227"/>
         <source>exist, are you sure replace?</source>
-        <translation type="unfinished"></translation>
+        <translation>이미 존재하니, 바꾸겠는가?</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="1920"/>
         <source>NewFolder</source>
-        <translation type="unfinished"></translation>
+        <translation>새 폴더</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="2208"/>
         <source>Undo</source>
-        <translation type="unfinished"></translation>
+        <translation>되돌리기</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="2215"/>
         <source>Redo</source>
-        <translation type="unfinished"></translation>
+        <translation>다시실행</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="2414"/>
         <source>warn</source>
-        <translation type="unfinished"></translation>
+        <translation>경고</translation>
     </message>
     <message>
         <location filename="../qt5-ukui-filedialog/kyfiledialog.cpp" line="2414"/>
         <source>This operation is not supported.</source>
-        <translation type="unfinished"></translation>
+        <translation>이 조작은 지원되지 않습니다.</translation>
     </message>
 </context>
 </TS>
